central-pet-front/.env.example

```
@@ -8,3 +8,6 @@ VITE_AUTH_STRATEGY=mock
 
 # Exibe gate de escolha mock na interface (true | false)
 # VITE_DISPLAY_MOCK_CHOICE_GATE=false
+
+# Exibe o nome do site
+VITE_SITE_NAME=Centrau Pet
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)